### PR TITLE
Small fixes of minetest.has_feature

### PR DESCRIPTION
**Pull request by Rui914**

### CODE REVIEW

**Comment by PilzAdam** [comment]
Looks good.

File: builtin/game/features.lua [hunk 1]
```diff
@@ -12,9 +12,9 @@ core.features = {
 
 function core.has_feature(arg)
 	if type(arg) == "table" then
-		missing_features = {}
-		result = true
-		for ft, _ in pairs(arg) do
+		local missing_features = {}
+		local result = true
+		for ftr in pairs(arg) do
```
Comment (rubenwardy) [inline builtin/game/features.lua]: is this not meant to be ftr, _?

Comment (PilzAdam) [inline builtin/game/features.lua]: @rubenwardy the second return value would always be `true`, no need to store it in a temporary variable.

Comment (Rui914) [inline builtin/game/features.lua]: It's shortened.
It'll work without problems.

**Comment by est31** [comment]
d4938554818aa71aceb0d360f33c6cfd42f67008

File: builtin/game/features.lua (hunk 1 above)
Comment (rubenwardy) [inline builtin/game/features.lua]: I think I got confused with Python, where ftr would = a tuple (name, value)

**Pull request closed without merging**